components/kubeflow/deployer/component.yaml

```diff
@@ -6,6 +6,7 @@ inputs:
   - {name: Namespace,     type: String, default: 'kubeflow',      description: 'Kubernetes namespace where the TS-serving service should be deployed.'}
   - {name: Server name,   type: String, default: 'model-server',  description: 'TF-serving server name to use when deploying.'}
   - {name: PVC name,      type: String, default: '' ,             description: 'Optional PersistentVolumeClaim to use.'}
+  - {name: Service Type,  type: String, default: '' ,             description: 'Optional Service type to use, two options: "ClusterIP"(default if not set) and "NodePort".'}
```

Comment:
https://travis-ci.com/kubeflow/pipelines/jobs/192099856#L1422
* "Service Type" -> "Service type"
* Probably add space before parenthesis here: "two options: "ClusterIP"(default if not set)"

Reply:
Thanks, I will enhance it base on your comment.

Reply:
JFYI: The first one ("Service Type" instead of "Service type") was a bug. Without fixed name the definition won't load, because you use the name later in the spec. (see the failed test https://travis-ci.com/kubeflow/pipelines/jobs/192099856#L1422).